package.json

```diff
@@ -42,6 +42,7 @@
     "@types/react-transition-group": "2.0.16",
     "@types/react-virtualized": "9.18.12",
     "@types/react-window": "1.7.0",
+    "@types/remarkable": "^1.7.4",
```

Comment:
Use fixed versions in package.json, ie 1.7.4 without the ^ - and don't forget to update the lock-file